lib/SPIRV/libSPIRV/SPIRVOpCode.h

```diff
@@ -62,7 +62,9 @@ inline bool isAtomicOpCode(Op OpCode) {
   static_assert(OpAtomicLoad < OpAtomicXor, "");
   return ((unsigned)OpCode >= OpAtomicLoad &&
           (unsigned)OpCode <= OpAtomicXor) ||
-         OpCode == OpAtomicFlagTestAndSet || OpCode == OpAtomicFlagClear;
+         OpCode == OpAtomicFlagTestAndSet || OpCode == OpAtomicFlagClear ||
```

Comment:
`atomic_flag_test_and_set/clear` aren't part of OpenCL 1.2 tho (https://www.khronos.org/registry/OpenCL/sdk/1.2/docs/man/xhtml/atomicFunctions.html and https://www.khronos.org/registry/OpenCL/sdk/2.0/docs/man/xhtml/atomic_flag_test_and_set.html ) (just challenging PR name correctness). Also as you aren't changing any tests appropriate to `atomic_flag_test_and_set/clear` instructions it makes me thinking, that they are either already being translated without your change, or they aren't being translated in the translator at all.